modules/ROOT/pages/bonita-bpm-overview.adoc

```diff
@@ -1,12 +1,23 @@
-= Bonita platform overview
+= Bonita components
 :description: Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
 
 Bonita is an open-source and extensible platform for business process automation and optimization. The Bonita platform accelerates development and production with clear separation between capabilities for visual programming and for coding.  BPMN graphical notation, full extensibility, and reusable components allow smooth collaboration among the different profiles on the IT team, and with the business teams. Bonita integrates with existing information systems, orchestrates heterogeneous systems, and provides deep visibility into processes across the organization.
 
 The Bonita Platform has two components: the development environment, Bonita Studio, and the runtime environment, Bonita Runtime.
 
+[.card-section]
 == Bonita Studio
 
+[.card.card-index]
+--
+xref:bonita-studio.adoc[[.card-title]#Bonita Studio# [.card-body.card-content-overflow]#pass:q[After reading what follows, you may want to know more]#]
+--
+
+[.card.card-index]
+--
+xref:ui-designer-overview.adoc[[.card-title]#Bonita UI Designer# [.card-body.card-content-overflow]#pass:q[The other component within Bonita Studio]#]
+--
+
 Bonita Studio contains all the elements needed to develop and compile a Bonita application. It is a graphical environment for creating processes, applications, data models, and users views (pages and forms). It contains three major design tools:
```

Comment:
This is not true anymore, extensions must be developed outside the Bonita Studio. 

We also talk a lot about the 'Bonita Portal' embedded in the Bonita Studio. It doesn't exist anymore, all this paragraph should be updated.

Reply:
fixed